src/Navigation.js

```diff
@@ -15,6 +15,7 @@ class Navigation extends Component {
 
             <Nav className="mr-auto"></Nav>
             <Nav.Link><Link to="/signup">Sign up</Link></Nav.Link>
+            <Nav.Link><Link to="/signin">Sign in</Link></Nav.Link>
```

Comment:
Please change it to <Nav.Link href="/signup"></Nav.Link>. I have changed it last night before you commit